images/c4-container.puml

```diff
@@ -0,0 +1,48 @@
+@startuml c4-container
+!theme seedcase from https://raw.githubusercontent.com/seedcase-project/seedcase-theme/main
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+Person(adm, "Administrator")
+Person(inp, "Data Contributor")
+Person(req, "Data Requester")
+
+Person_Ext(ext, "External User")
+
+Container_Boundary(seed, "Seedcase Data Resource", "web-interface") {
+	Container(land, "Main Seedcase page", "navigation page", "Main page showing installed apps")
```

Comment:
@lwjohnst86 How do you imagine users navigating the different apps?

Reply:
As discussed we'll stick with the landing page for now.